django_spanner/operations.py

```diff
@@ -133,13 +251,32 @@ def get_db_converters(self, expression):
             converters.append(self.convert_uuidfield_value)
         return converters
 
+    # TODO: here and below, remove unused expression and connection parameters
+    # or include them to the code.
```

Comment:
@mf2199 I moved this comment in https://github.com/googleapis/python-spanner-django/pull/564/commits/d36b9abd628dc2fd4a2d4535583d8dfc4ada3b73 since the preceding change had this comment in each of the converter docstrings:

> #TODO remove unused expression and connection parameters or include them to the code.

But converters are part of the django db API, and we can't change the signature without also replacing django internals. See e.g. https://github.com/django/django/blob/8093aaa8ff9dd7386a069c6eb49fcc1c5980c033/django/db/models/sql/compiler.py#L1084.